src/processor/operator/ddl/CMakeLists.txt

```diff
@@ -2,7 +2,10 @@ add_library(kuzu_processor_operator_ddl
         OBJECT
         create_node_table.cpp
         create_rel_table.cpp
-        drop_property.cpp)
+        ddl.cpp
+        drop_property.cpp
+        drop_table.cpp
+        )
```

Comment:
i think our convension is not to add a new line between the last file and the last bracket.ddl.cpp
```
        drop_property.cpp
        drop_table.cpp)
```